digitalocean/Droplet.py

```diff
@@ -639,5 +640,15 @@ def get_kernel_available(self):
 
         return kernels
 
+    def update_volumes_data(self):
+        """
+            Trigger refresh of volumes table
+        """
+        self.volumes = list()
+
+        for volume in self.volume_ids:
+            self.volumes.append(Volume().get_object(self.token, volume))
```

Comment:
can we make it a little more readable and KISS?
One line is not a style I want

Reply:
Understandable :)